### ISSUE TITLE
Waybar crashes on launch

### ISSUE THREAD

**Issue by a-rennie** [Alexays/Waybar#2724]
After a recent system update, waybar crashes on launch with the error: `waybar: symbol lookup error: waybar: undefined symbol: _ZN6spdlog7details7log_msgC1ENS_10source_locEN3fmt2v917basic_string_viewIcEENS_5level10level_enumES6_`.
I suspect this is due to the recent update of `libspdlog1_12` from `3.1` - `3.2`
System specs:
```
OS: openSUSE 20231203
Kernel: x86_64 Linux 6.6.3-1-default
Packages: 2770
Shell: zsh 5.9
Resolution: 1920x1080
WM: sway
CPU: Intel Core i7-10510U @ 8x 4.9GHz
GPU: Mesa Intel(R) UHD Graphics (CML GT2)
```

**Comment by drunckj** [comment]
Having the same issue in Opensuse Tumbleweed.

**Comment by TheM1Stery** [comment]
I have the same issue here on opensuse. Which library is missing?

**Comment by drunckj** [comment]
It starts working if you downgrade libspdlog1_12.

**Comment by drunckj** [comment]
downgrade libspdlog1_12 to an older version and it will work fine.

**Comment by panda3045** [comment]
> It starts working if you downgrade libspdlog1_12.

If anyone is looking for the previous (working) version, you can find it [here](http://download.opensuse.org/history/20231204/tumbleweed/repo/oss/x86_64/libspdlog1_12-1.12.0-3.1.x86_64.rpm)

**Comment by panda3045** [comment]
issue appears to be fixed for me on 3.2 now
